tests/core/test_broker.py

```diff
@@ -310,6 +311,23 @@ async def test_no_pika_logs_if_no_debug_mode(caplog: LogCaptureFixture):
         for record in caplog.records
     )
 
+    broker = PikaEventBroker(
+        "host", "username", "password", retry_delay_in_seconds=1, connection_attempts=1
+    )
+
+
+def test_raise_exception_port(monkeypatch: MonkeyPatch):
+
+    with pytest.raises(RasaException):
+        PikaEventBroker(
```

Comment:
I'm aware you cannot see the full stacktrace from Sentry, not sure why it got clipped, but I would actually use exactly what gets called further up in the stacktrace: the EventBroker factory from [here](https://github.com/RasaHQ/rasa/blob/51185ee5e66404864239b567dd67b34f77579fce/rasa/core/brokers/broker.py#L20).
You can create an instance of EndpointConfig which you can pass as argument to the broker factory:
`EndpointConfig(username="username", password="password", type="pika", port="PORT")

Reply:
In addition, note of caution - the EventBroker factory is async so you'll need to use the keyword `await` when calling it. As well as making the test async too.

Reply:
ok, i'll do this

Reply:
@ancalita can you check if it is right now?

Reply:
@jppgomes Sure, you could also re-request review by clicking on the arrows-in-a-circle icon to the right of the picture avatar under Reviewers section (top right).  
Does the test pass?
Please also add this assertion check in the test:
```
with pytest.raises(RasaException) as e:
     ...

assert "Port could not be converted to integer." in str(e.value)
```